docs/hsi.md

```diff
@@ -392,6 +392,28 @@ To meet HSI-2 on systems that run this test, the result must be `locked`. *[v1.5
 
 - [Intel Direct Connect Interface](https://www.intel.co.uk/content/www/uk/en/support/articles/000029393/processors.html)
 
+<a id="org.fwupd.hsi.Tpm.EmptyPcr"></a>
+
+### [Empty PCR in TPM](#org.fwupd.hsi.Tpm.EmptyPcr)
+
+The system firmware is responsible for measuring values about its boot stage in PCRs 0 through 7.
+Some firmwares have bugs that prevent them from measuring some of those values, breaking the fundamental assumption of the Measured Boot chain-of-trust.
+
+**Impact:** A local attacker could measure fake values into the empty PCR, corresponding to a firmware and OS that do not match the ones actually loaded.
```

Comment:
Not necessarily: in the case of this bug, the TPM event log (for that register) would be empty as well. The bug is the firmware doesn't measure anything when it should. There's no reason to think it would correctly set the eventlog when it's not measuring. (I didn't check with the impacted firmwares though.)

Reply:
Got it thanks.